man/systemd.mount.xml

```diff
@@ -366,6 +366,18 @@
         <varname>Options=</varname> setting in a unit file.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>x-systemd.pcrfs</option></term>
+
+        <listitem><para>Measures file system identity information (mount point, type, label, UUID, partition
+        label, partition UUID) into PCR 15 after the file system has been mounted. This ensures the
+        <citerefentry><refentrytitle>systemd-pcrfs@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        or <filename>systemd-pcrfs-root.service</filename> services are pulled in by the mount unit.</para>
+
+        <para>Note that this option can only be used in <filename>/etc/fstab</filename>, and will be ignored
+        when part of the <varname>Options=</varname> setting in a unit file.</para></listitem>
```

Comment:
How does this mesh with the discoverable partition spec, e.g. on a system without fstab?

Reply:
gpt-auto-generator also hooks this up automatically, see the gpt-auto-generator patch in this PR

Reply:
(will add comment to man page saying this explicitly)

Reply:
@poettering MEETING